nmrglue/analysis/integration.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+
+
+def integrate(data, unit_conv, limits, unit='ppm', noise_limits=None,
+              norm_to_range=None, calibrate=1.0):
+    """
+    Integrate one 1D data array within limits given in units. Data points must
+    be equally spaced.
+
+    Functional form of integration is:
+
+    .. math::
+        value = \sum_a^b s(x_{i}) dx
+
+    Where:
+    s is the signal, a and b are the limits of integration and dx is the width
+    of each bin.
+
+    The integration error due to baseline noise is calculated as:
+
+    ..math::
+        error = \sigma_{vol} = \sigma \sqrt{n},
+
+    if the noise_limits are set.
+
+    Where:
+    .. math::
+        n = \frac{|b-a|}{dx}+1
```

Comment:
n = \\frac{|b-a|}{dx}+1

Apparently \f is the line feed symbol and needs to be escaped.